### DISCUSSION TITLE
📣 POLL: Have you used sample code from this repository in your application?

### DISCUSSION THREAD

**Discussion by rodrifed** [amzn/selling-partner-api-samples#42] [Polls]
The SP-API team is looking for your feedback in order to identify the resources that would help you the most in your API integration journey.
Please answer the poll below, raise your voice, and help us shape the future of SP-API content!

**Comment by johnkw** [comment]
No, just came here looking for any hint on how to get past the `FBA_INB_0182` blocker but couldn't find anything.
